### ISSUE TITLE
I don't know if you can clarify

### ISSUE THREAD

**Issue by gallegonovato** [wwmm/easyeffects#1894]
Good afternoon:

First of all I apologize if this is not the correct way to contact you. But I have some doubts about helping in the translation of your application.

More specifically with these sentences:

https://hosted.weblate.org/translate/easyeffects/main/es/?q=state:%3Ctranslated

Since Left Wet Level for example literally translates to: Nivel de humedad izquierdo

But being an equalizer application, I think it could be put: Sound level on the left side

But I don't know if I am on the right track or if I could change or explain a little more the meaning you are looking for.

Best regards and sorry for the inconvenience.

**Comment by wwmm** [comment]
Hi. Thanks for helping with the translations. It may seem strange but you have to do the literal translation. It is the way people that work with digital signal processing talk about this. "Sound Level" is not going to provided the correct meaning. IT has to be the literal meaning of "wet" and "dry".

**Comment by gallegonovato** [comment]
Good afternoon @wwmm

No need to say thank you. In the languages I can, I will gladly do it :)

Thanks for the clarification. I didn't know that, that they had those exact words. That's why the dry or wet on the right sounded strange to me.

I apologize for this mistake. So knowing what you are telling me I can translate better.

A hug and thanks a million for your help and time.

**Comment by gallegonovato** [comment]
Good afternoon @wwmm

I am commenting here so as not to disturb you in a new thread.

I don't know if you could add or clarify a bit more this string to translate https://hosted.weblate.org/translate/easyeffects/main/es/?q=check:translated

And this one I think it is better to put so as not to be modified https://hosted.weblate.org/translate/easyeffects/main/es/?q=has:dismissed-check

Best regards and thanks for your time

**Comment by wwmm** [comment]
>-inf

This one means ` - infinity`. Depending on the translation difficulty putting the whole `infinity` word is a decent alternative.

> Herm Wide

The `Herm` refers to `Hermite interpolated cubic function`. Depending on the language the name `Hermite` may be written in slightly different ways. So it is worth to see how people write it in your language. The word `Wide` and the others that come after `Hermite` can be translated too. If I am not mistaken they are related to the shape of the Hermite function being used.

**Comment by gallegonovato** [comment]
Good afternoon @wwmm

I had doubts in the first inf. if you meant: infinite, file not found, interface, info, etc.... That's why I preferred to ask before putting my foot in my mouth.

Then Herm Wide as you capitalize it I also thought of someone's name. Although wide can be wide.

Then Herm apart from a country or island. It is also a capacitor. And also if I am not mistaken it also refers to a stone statue. Hence my doubts, since I thought when I saw it in capital letters that it could be a name of someone and maybe if I translated it I would make a mistake. Since Herm also refers to Hermes. But I missed the Wide , since here it is: Ancho

A hug and thanks for your help and time.
